lib/rex/post/meterpreter/client_core.rb

```diff
@@ -431,7 +431,7 @@ def get_ssl_hash_verify
   # Migrates the meterpreter instance to the process specified
   # by pid.  The connection to the server remains established.
   #
-  def migrate(pid, writable_dir = nil)
```

Comment:
The new calling syntax also must have the :target_pid and :timeout key, it would also be nice to document these too.

```ruby
# Correct way of calling the new #migrate method
client.core.migrate(pid: target_pid, timeout: 60)
```